imports/client/components/CallLinkBox.tsx

```diff
@@ -0,0 +1,345 @@
+      this.connectSpectrogramIfReady();
+    } else {
+      // Ignore non-audio tracks.
+      // this.remoteStream.addTrack(e.track);
+    }
+
+    if (this.audioRef.current) {
+      this.audioRef.current.srcObject = this.remoteStream;
+    }
+  };
+
+  onNegotiationNeeded = (_e: Event) => {
+    // If we're the initiator, get the ball rolling.  Create an
+    // offer, so we'll:
+    // 1) generate an SDP descriptor and
+    // 2) start doing STUN to collect our ICE candidates.
+    //
+    // We must make the offer *after* we have a user media stream;
+    // browsers won't bother if the peer doesn't have a stream worth
+    // sharing, because SDP requires knowing what the stream format
+    // is.  (This is fine; we already have the stream by the time we
+    // construct a CallLink.)
+    if (this.isInitiator) {
+      this.pc.createOffer().then(this.onLocalOfferCreated);
+    }
+
+    // this.log('negotiationNeeded', e);
+  };
+
+  onIceConnectionStateChange = (_e: Event) => {
+    // this.log('new ice connection state change:');
+    // this.log(e);
+    this.setState({
+      iceConnectionState: this.pc.iceConnectionState,
+    });
+  };
+
+  connectSpectrogramIfReady = () => {
+    // Connect the stream to the spectrogram if ready.
+    if (this.spectrumComponent && this.spectrumStream) {
+      this.spectrumComponent.connect(this.spectrumStream);
+    }
+  };
+
+  render() {
+    const name = (this.props.peerProfile && this.props.peerProfile.displayName) || 'no profile wat';
+    return (
+      <OverlayTrigger
+        key={`viewer-${this.props.peerParticipant._id}`}
+        placement="right"
+        overlay={(
+          <Tooltip
+            id={`caller-${this.props.peerParticipant._id}`}
+            className="chatter-tooltip"
+          >
+            <div>{name}</div>
+            <div>
+              connection status:
+              {' '}
+              {this.state.iceConnectionState}
+            </div>
+            {this.props.peerParticipant.muted &&
+              <div>Muted (no one can hear them)</div>}
+            {this.props.peerParticipant.deafened &&
+              <div>Deafened (they can&apos;t hear anyone)</div>}
+          </Tooltip>
+        )}
+      >
+        <div
+          key={`viewer-${this.props.peerParticipant._id}`}
+          className={classnames('people-item', {
+            muted: this.props.peerParticipant.muted,
+            deafened: this.props.peerParticipant.deafened,
+            live: !this.props.peerParticipant.muted && !this.props.peerParticipant.deafened,
+          })}
+        >
+          <span className="initial">{name.slice(0, 1)}</span>
+          <div className="webrtc">
+            {this.props.peerParticipant.muted && <span className="icon muted-icon"><FontAwesomeIcon icon={faMicrophone} /></span>}
+            {this.props.peerParticipant.deafened && <span className="icon deafened-icon"><FontAwesomeIcon icon={faHeadphonesAlt} /></span>}
```

Comment:
FWIW I find the iconography here a little odd. I would expect to see [`microphone-slash`](https://fontawesome.com/icons/microphone-slash) for muted and...well, I guess it would be ideal if there was a `headphones-alt-slashed` for deafened. The color on its own doesn't indicate mute/deaf

Reply:
Maybe [`volume-slash`](https://fontawesome.com/icons/volume-slash)?

Reply:
volume-slash is only available in Pro.

I'll swap to microphone-slash and [volume-mute](https://fontawesome.com/icons/volume-mute?style=solid)